tests/plenary/object/calendar_spec.lua

```diff
@@ -0,0 +1,87 @@
+local helpers = require('tests.plenary.helpers')
+local Calendar = require('orgmode.objects.calendar')
+local Date = require('orgmode.objects.date')
+
+local close_all_buffers = function()
+  vim.cmd([[silent! %bw!]])
+end
+
+local feed = function(keys)
+  vim.api.nvim_feedkeys(vim.api.nvim_replace_termcodes(keys, true, true, true), 'x', true)
+end
+
+local mappings = require('orgmode.config.defaults').mappings
+local prefix = mappings.prefix
+local open_calendar = mappings.org.org_deadline:gsub('<prefix>', prefix)
+
+describe('Calendar', function()
+  after_each(close_all_buffers)
+  it('should open a calendar at plain headline', function()
+    helpers.create_agenda_file({
+      '#+TITLE: Test',
+      '',
+      '* Some Headline',
+    })
+    vim.fn.cursor(3, 1)
+    feed(open_calendar)
```

Comment:
Let's use the real mapping here via `vim.cmd` like it's done for some of the [other tests](https://github.com/nvim-orgmode/orgmode/blob/c9bf6d8e926c5d6ad7103ee8e7b1e38500b7bfc5/tests/plenary/ui/mappings/headline_spec.lua?plain=1#L265)
```suggestion
    vim.cmd('norm ,oid')
```
That way you can also remove the 3 variables above the describe.

Reply:
Exchanged it. I still need the feed method to actually simulate keystrokes within the calendar. This didn't work with `vim.cmd`.